### ISSUE TITLE
[Feature Request] Change Instant Search into a full-feature search bar

### ISSUE THREAD

**Issue by primehalo** [notepad-plus-plus/notepad-plus-plus#14046]
### Description of the Issue
I would love it if the Instant Search could be made to have all the search & replace options as the regular search window dialog (Whole Word Only, Regular Expression, Count, Find All, Find in Files, Replace, etc). RJ TextEd has this and I've found it to be very useful.

**Comment by alankilborn** [comment]
Instant Search -> _Incremental Search_ (presumed)

**Comment by primehalo** [comment]
Yep, thanks, I fixed it in the description. I don't see a way to update the title.

**Comment by alankilborn** [comment]
The title is editable; here's one of my open issues so I can show you how to edit yours:

![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/77065706/5e959f6f-bb39-44be-a41d-a84e1804d434)

**Comment by primehalo** [comment]
Thanks, got the title changed as well.
